### ISSUE TITLE
Mouse and keyboard do not work when game starts after cutscene

### ISSUE THREAD

**Issue by Costava** [codyebberson/js13k-minipunk#1]
When the game starts after the cutscene, the mouse and keyboard do not work.
No errors or warnings are seen in the console.

Workaround:
1. Press escape
2. Click "Continue"
3. Click the mouse again
4. The mouse and keyboard controls now work as expected

Firefox 79.0 and 80.0.1 (both 64-bit)

Unable to reproduce in Chromium 85.0.4183.102 64-bit

**Comment by codyebberson** [comment]
Yes, I've noticed this on recent versions of Edge too.  The root cause is inconsistent behavior in [Pointer Lock API](https://developer.mozilla.org/en-US/docs/Web/API/Pointer_Lock_API).  In particular, Chrome appears to lock the cursor immediately, and other browsers do it on a delay.
